Automattic/newspack-popups
# Addthe Importer

### All Submissions:






### Changes proposed in this Pull Request:



Creates the Importer class

**This PR is depends on https://github.com/Automattic/newspack-popups/pull/1021**

Important: For now, we are not importing Categories and Tags. This will be handled in future PRs... The idea is to give the user the option to map tags and categories to existing terms or create them...

### How to test the changes in this Pull Request:

1. Tests are passing?

This will be easier to test once we implement the CLI commands for the exporter and the importer, but if you really want to test:

* Use the exporter to generate some data
* `json_encode` the data exported and save it to a file
* Delete all campaigns, prompts and segments from the site
* Load the contents of the file, `json_decode` it and import it using the Importer class
* Confirm that everything is created as expected


### Other information:
